WordPress/Sniffs/PHP/YodaConditionsSniff.php

```diff
@@ -20,11 +20,28 @@
 class WordPress_Sniffs_PHP_YodaConditionsSniff extends WordPress_Sniff {
 
 	/**
+	 * The tokens that indicate the start of a condition.
+	 *
+	 * @since 0.12.0
+	 *
+	 * @var array
+	 */
+	protected $condition_start_tokens;
+
+	/**
 	 * Returns an array of tokens this test wants to listen for.
 	 *
 	 * @return array
 	 */
 	public function register() {
+
+		$this->condition_start_tokens  = PHP_CodeSniffer_Tokens::$booleanOperators;
+		$this->condition_start_tokens += PHP_CodeSniffer_Tokens::$assignmentTokens;
+		$this->condition_start_tokens[ T_CASE ] = T_CASE;
+		$this->condition_start_tokens[ T_RETURN ] = T_RETURN;
+		$this->condition_start_tokens[ T_SEMICOLON ] = T_SEMICOLON;
+		$this->condition_start_tokens[ T_OPEN_PARENTHESIS ] = T_OPEN_PARENTHESIS;
```

Comment:
I'd personally align the whole list, but also initialize it as a local variable only to assign it to the property at the end so the lines don't get as long.

```php
		// Create the condition starters array only once.
		$starters                       = PHP_CodeSniffer_Tokens::$booleanOperators;
		$starters                      += PHP_CodeSniffer_Tokens::$assignmentTokens;
		$starters[ T_CASE ]             = T_CASE;
		$starters[ T_RETURN ]           = T_RETURN;
		$starters[ T_SEMICOLON ]        = T_SEMICOLON;
		$starters[ T_OPEN_PARENTHESIS ] = T_OPEN_PARENTHESIS;

		$this->condition_start_tokens = $starters
```